transform/mattermost-analytics/models/marts/web_app/platform/_platform__models.yml

```diff
@@ -0,0 +1,90 @@
+version: 2
+
+models:
+  - name: eg_performance_events
+    description: |
+      Events Group performance_events table containing metrics 
+      from web-app and web-app RC telemetry for category = performance.
+
+    columns:
+      - name: _source_relation
+      - name: context_page_referrer          
+      - name: channel                        
+      - name: context_app_namespace          
+      - name: id
+      - name: user_actual_id                
+      - name: context_library_name           
+      - name: type                           
+      - name: context_app_version            
+      - name: user_actual_role               
+      - name: context_page_url               
+      - name: context_os_name                
+      - name: context_page_title             
+      - name: anonymous_id                   
+      - name: context_app_build              
+      - name: context_page_search            
+      - name: context_library_version        
+      - name: context_ip                     
+      - name: context_useragent              
+      - name: context_app_name               
+      - name: context_locale                 
+      - name: context_screen_density         
+      - name: context_page_path              
+      - name: context_os_version             
+      - name: category                       
+      - name: duration                     
+      - name: num_of_request                 
+      - name: max_api_resource_size          
+      - name: longest_api_resource_duration  
+      - name: user_id                        
+      - name: userid                         
```

Comment:
Looks like a typo in the implementation. Perhaps a `coalesce` between these two can simplify things.

Reply:
I removed the field, there is no data in the field, I will remove the field in Looker as well, after this is merged. There are some changes I need to make to the Looker view.